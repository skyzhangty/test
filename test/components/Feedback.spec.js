import React from 'react';
import renderer from 'react-test-renderer';

import Feedback from '../../src/scripts/components/Feedback';

describe('components', () => {
	describe('Feedback', () => {
		it('should render self', () => {
			const component  = renderer.create(
				<Feedback feedback={{title: 'title', message: 'message'}}/> );

			const tree = component.toJSON();
			expect(tree).toMatchSnapshot();
		});
	});
});

