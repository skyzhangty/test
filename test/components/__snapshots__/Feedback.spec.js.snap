// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Feedback should render self 1`] = `
<div
  className="alert-info"
  id="divFeedbackBlock"
>
  <h4
    id="h4Title"
  >
    title
  </h4>
  <p
    id="pMessage"
  >
    message
  </p>
</div>
`;
